### DISCUSSION TITLE
投稿者用のリリースノートディスカッションエリアを作って欲しい

### DISCUSSION THREAD

**Discussion by fygar256** [increments/qiita-discussions#950] [要望・アイデア・不具合]
投稿者用のリリースノートがあれば良いと思います。Qiitaに作るのはシステムを改造しなくてはいけないので手間がかかるから、GitHub上でもいいと思います。普通、記事のアップデートはあまりしないと思います故に、リリースノートに書かれるエントリも少ないと思いますので、アップデートする人用に作って欲しいです。広告用にあればいいと思います。記事を書き換えたらタイムラインのトップに上げるというアイディアは諸事情のため難しそうですので、GitHub上に作れば、邪魔にならなくて済むと思うし、あれば、最新情報をチェックする事ができるようになるので。最近はコードを投稿する人が少なくなったので、少数派の意見となりそうですが。
我儘なお願いかも知れませんが、ご検討をよろしくお願いします。Regards.

**Comment by acronhub** [comment]
@fygar256
ご意見ありがとうございます！

自身が作成したソフトウェアの説明と更新内容がわかるようなページを作成したいというご要望であっていますでしょうか？

もしご要望があっていた場合ですが、
@fygar256 さんのいうようにQiitaの機能としては、今のところ未定です 🙇  

提案になりますが、GitHub Pagesなどの外部サービスを利用して、ソフトウェアの説明や更新内容をまとめたページを作成するのはいかがでしょうか？

ご検討いただければ幸いです 🙏

**Reply by fygar256** [reply]
どうもご返答ありがとうございます。検討してみます。やはり、コードを投稿する人は少ないのでしょうか。寂しい限りです。
こういう要望もあるということで、オープンのままにしておきます。
